README.md

```diff
@@ -43,15 +43,15 @@ This will clone the repo to the `master` branch, which contains code for latest
 and hot-fixes.
 
 ```
-git clone --recursive -b master git@github.com:clara-genomics/ClaraGenomicsAnalysis.git
+git clone --recursive -b master https://github.com/clara-genomics/ClaraGenomicsAnalysis.git
```

Comment:
@edawson can you fix this branch as well? it looks like it got some changes from `master` that have already been merged into `dev-v0.5.0`. rest of it looks good to me!